### DISCUSSION TITLE
Best way to pair an Yi-Hack camera with Yi App?

### DISCUSSION THREAD

**Discussion by chrbar11** [roleoroleo/yi-hack-Allwinner-v2#980] [General]
Hello,

I've a camera hacked with Yi-Hack which has never been paired with Yi App.
What is the best way to pair it with Yi App now?

Do I have just to use the reset hole/pin on camera (as discribed on the below video) to clean the configuration and restart the pairing process?
https://help.yitechnology.com/hc/en-us/articles/360045037674-How-to-reset-your-YI-Home-camera-

If yes, will it break the Yi-Hack on this camera, and I'll have I to reinstall Yi-Hack?

Thanks a lot for your help :)
Chris

**Comment by roleoroleo** [comment]
Yes, the reset pin is the best way.
The hack will survive to this reset.
If you prefer, you can restart the cam without the sd card, before the reset.

**Reply by chrbar11** [reply]
Thank you roleo :)
